skops/io/_visualize.py

```diff
@@ -202,9 +204,6 @@ def walk_tree(
     node: :class:`skops.io._audit.Node`
         The current node to visit. Children are visited recursively.
 
-    show: "all" or "untrusted" or "trusted"
-        Whether to print all nodes, only untrusted nodes, or only trusted nodes.
-
```

Comment:
probably removed by mistake?

Reply:
Looked like an orphan comment when updating the docstring (`walk_tree` has no parameter `show`).